README.md

```diff
@@ -2,31 +2,86 @@
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| aws | n/a |
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| account\_alias | The desired AWS account alias. | `string` | n/a | yes |
+| region | AWS region. | `string` | n/a | yes |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| dynamodb\_table | The name of the dynamo db table |
+| logging\_bucket | The logging\_bucket name |
+| state\_bucket | The state\_bucket name |
+
+<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
 ## Bootstrapping
 
-Copy the code in this repo to where your Terraform config will live. For example, it could live in a directory like `terraform/account-alias/bootstrap`.
+Create a new directory where your Terraform config will live. For example, it could live in a directory like `terraform/account-alias/bootstrap`.
 
 If your `aws` commands run via [aws-vault](https://github.com/99designs/aws-vault) and you are using root credentials, you'll need to set the `--no-session` flag so the IAM operations can run without being MFA'd. If you're using the [Truss aws-vault-wrapper](https://github.com/trussworks/terraform-layout-example/blob/master/bin/aws-vault-wrapper) you can set the `AWS_VAULT_NO_SESSION` environment variable. If you don't do this you'll receive an `InvalidClientTokenId` error.
 
 If you are running your `aws` commands via [aws-vault](https://github.com/99designs/aws-vault) and are using a role assumption, you will want to run this script without `AWS_VAULT_NO_SESSION`. The role assumption between different profiles requires session behavior. Additionally, this script will attempt to run `aws s3 ls` before checking for bucket existence so that you can create a session token that may require an MFA handshake.
 
-Run the `bootstrap` script, specifying your AWS account alias and region:
+Set up your `bootstrap/main.tf` file to look like the example above. Don't forget to include a `providers.tf` file that looks like this:
 
-```sh
-./bootstrap my-account-alias region
+```hcl
+provider "aws" {
+  version = "~> 2.70.0"
+  region  = var.region
+}
+
+provider "template" {
+  version = "~> 2.1.2"
+}
 ```
 
-The account alias will be configured (if not set), the resources will be created, and a git commit will be made with the tfvars and state files. Push those changes to your repo.
+Then run this like any other terraform module:
 
-## Pre-existing environments
+```sh
+terraform init && terraform plan
+# Review output of plan
+terraform apply
+```
 
-The code can be customized for specific environments. For example, if you're bootstrapping Terraform into an existing environment, you may want to use an existing logging bucket for the S3 logs. Edit the Terraform and bootstrap script as you need.
+The account alias will be configured (if not set), the resources will be created, and a git commit will be made with the tfvars and state files. Push those changes to your repo.
```

Comment:
This won't create a git commit... I think you'll have to add instructions to specifically add/commit that and note that this should be the only tfstate file committed to your terraform repo as the bootstrapped s3 bucket and dynamodb table should be used to handle the other backends.